### DISCUSSION TITLE
send email when i create a new user

### DISCUSSION THREAD

**Discussion by samaremad** [serenity-is/Serenity#5479] [Help]
hi everyone 

i want when i create a user in users table by using real email address send an email to that address how can i do that?

**Comment by Zahar661** [comment]
Here is an example of forming a letter. but you need to initiate in  UserRepository.cs MySaveHandler  AfterSave() 
https://github.com/serenity-is/Serene/blob/master/Serene/Serene.Core/Modules/Membership/Account/ForgotPassword/AccountPage.ForgotPassword.cs

`Common.EmailHelper.Send(emailSubject, emailBody, user.Email);`

**Reply by samaremad** [reply]
PLEEEEEEEEEEEEESE HELP ME 
THIS EXAMPLE NOT CLEAR TO ME
AND IT IS REALLY IMPORTANT TO DO IT NOW 
HOW CAN I DO IT 
I NOW I MUST ADD CODE IN  UserRepository.cs MySaveHandler AfterSave()
BUT I DONT KNOW WHAT TO WRITE AFTER I CREATED THE USER

**Comment by Zahar661** [comment]
```
             if (IsCreate)
                {
                    Common.EmailHelper.Send("New user", "Hello!!!<br>You "+ Row.DisplayName+ ",<br> you're cool!!!", Row.Email);
                }
```

**Reply by Zahar661** [reply]
![image](https://user-images.githubusercontent.com/26430136/108025724-089e3e80-7038-11eb-94c8-33873f3af0e3.png)
